modules/admin-ui/src/main/resources/OSGI-INF/event_endpoint.xml

```diff
@@ -13,68 +13,42 @@
     <provide interface="org.osgi.service.cm.ManagedService"/>
   </service>
   <reference bind="setAdminUIConfiguration"
-             cardinality="1..1"
```

Comment:
I'm not sure how changes to the cardinality and policy in this file (and not others) relates to this ticket?

Reply:
I had to move and modify some of these anyway. The `1..1` is the default and technically unnecessary. I could move some of these changes to a separate pull request but it felt to be unnecessarily complex to do that with parts of them being modified anyway.